catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -150,6 +156,11 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.codice.test</groupId>
+            <artifactId>junit-extensions</artifactId>
+            <version>${codice-test.version}</version>
```

Comment:
✏️`<scope>test</scope>`

Reply:
Actually, the whole dependency shoudn't be required as it is already defined in the parent pom

Reply:
On 2.13.x it is not.

Reply:
Ah! True, this is what I will be doing with my own ticket. 😄